examples/demo-rollup/benches/rollup_coarse_measure.rs

```diff
@@ -26,12 +23,24 @@ use tempfile::TempDir;
 extern crate prettytable;
 
 use prettytable::Table;
-
-fn print_times(total: Duration, apply_block_time: Duration, blocks: u64, num_txns: u64) {
+use sov_modules_stf_template::TxEffect;
+
+fn print_times(
+    total: Duration,
+    apply_block_time: Duration,
+    blocks: u64,
+    num_txns: u64,
+    num_success_txns: u64,
+) {
     let mut table = Table::new();
 
     table.add_row(row!["Blocks", format!("{:?}", blocks)]);
     table.add_row(row!["Txns per Block", format!("{:?}", num_txns)]);
+    // subtract 1 because the first block contains only 1 txn which is the token create txn
```

Comment:
Not sure I follow this comment. Why do we exclude the first block?

Reply:
ah because the first block of the rng da is a block that includes only 1 transaction to create the token. the rest are as per the bench spec - 10k transfers

Reply:
As per discussion, I cleaned up this logic to completely exclude block 1 and the first token creation transaction, so we don't need to -1 in the counts and we also have more accurate timing